atd-vze/src/views/Crashes/personDataMap.js

```diff
@@ -107,6 +107,12 @@ export const primaryPersonDataMap = [
         label: "ZIP",
         editable: false,
       },
+      peh_fl: {
+        label: "Experiencing Homelessness",
+        editable: true,
+        format: "boolean",
```

Comment:
created this new "boolean" format because the "select" format uses lookup tables to determine what choices are in the drop down.. this new column doesn't have a lookup table, its not something we get from CRIS.

Reply:
No, I think you've made the perfect choice. It's exactly what it is - true/false to the person experiencing homelessness.

Reply:
Hmm thats a good point! I think since it's not in the section with the rest of the "Flags" it doesn't stand out as being abnormal, but I think changing to `YES` and `NO` does make sense from a UI perspective!